frontend/src/routes.js

```diff
@@ -32,6 +32,13 @@ app.get('/job-roles', async (req, res) => {
 
 });
 
+app.get('/bands', async (req, res) => {
+    const response = await fetch(api_url+'bands',{method:'GET',headers:{}})
+    const bandData = await response.json();
+    console.log({items:bandData});
```

Comment:
There's quite a lot of `console.log`s – these should be cleared up before pushing